gsl_projet/models.py

```diff
@@ -158,15 +158,22 @@ def get_or_create_from_ds_dossier(cls, ds_dossier: Dossier):
             )
         projet.address = ds_dossier.projet_adresse
 
-        projet_departement = (
-            ds_dossier.ds_demandeur.address.commune.departement
-            or ds_dossier.porteur_de_projet_arrondissement.core_arrondissement.departement
-        )
-        projet_arrondissement = (
-            ds_dossier.ds_demandeur.address.commune.arrondissement
-            or ds_dossier.porteur_de_projet_arrondissement.core_arrondissement
-        )
-        projet.demandeur, _ = Demandeur.objects.get_or_create(
+        projet_departement, projet_arrondissement = None, None
+        if ds_dossier.porteur_de_projet_arrondissement:
+            projet_departement = ds_dossier.porteur_de_projet_arrondissement.core_arrondissement.departement
+            projet_arrondissement = (
+                ds_dossier.porteur_de_projet_arrondissement.core_arrondissement
+            )
```

Comment:
```suggestion
           projet_arrondissement = ds_dossier.porteur_de_projet_arrondissement.core_arrondissement
            projet_departement = projet_arrondissement.departement
```
            
Ceci me semble faire moins de chemin pour les yeux :-]